monitor-template.md

```diff
@@ -0,0 +1,188 @@
+#test in github
+
+INFO TAB
+
+\<monitor logo>
+# \<Monitor name>
+
+Metadata associated with the \<name> collectd plugin can be found _here_. The relevant code for the plugin can be found _here_.
+
+- [Description](#description)
+- [Features](#features)
+- [Use](#use)
+- [Metrics](#metrics)
+- [Dimensions](#dimensions)
+
+## Description
+
+Monitors \<what> using the information provided by \<what> which collects metrics from \<what> instances by hitting these endpoints: 
+* <link to item>
+* <link to item>
+
+## Features
+
+For more information on the source data, see <https://...>
+
+Monitor Source Code \<link>
+
+Accepts Endpoints: <yes/no>
+
+Multiple Instances Allowed: <yes/no>
+
+
+## Use
+
+What the monitor is used for -- which metrics and for what -- discussion
```

Comment:
This sounds like what I would expect in Features, so I'm unclear on the difference between the info in these headings.

Reply:
Deleted Features heading. Usage is a new heading to get more usage info from the developer. Joe feels like they know more and must be given the space to describe usage.